install failure

```
❯ go get -v -u github.com/knqyf263/utern
github.com/knqyf263/utern (download)
github.com/aws/aws-sdk-go (download)
github.com/jmespath/go-jmespath (download)
github.com/briandowns/spinner (download)
github.com/fatih/color (download)
github.com/pkg/errors (download)
github.com/urfave/cli (download)
github.com/cpuguy83/go-md2man (download)
github.com/cpuguy83/go-md2man/vendor/github.com/shurcooL/sanitized_anchor_name
github.com/knqyf263/utern/cache
github.com/pkg/errors
github.com/knqyf263/utern/color
github.com/briandowns/spinner/vendor/golang.org/x/sys/unix
github.com/cpuguy83/go-md2man/vendor/github.com/russross/blackfriday/v2
github.com/cpuguy83/go-md2man/md2man
github.com/urfave/cli
github.com/briandowns/spinner/vendor/github.com/mattn/go-isatty
github.com/briandowns/spinner/vendor/github.com/mattn/go-colorable
github.com/briandowns/spinner/vendor/github.com/fatih/color
github.com/briandowns/spinner
github.com/knqyf263/utern/config
github.com/knqyf263/utern/cloudwatch
github.com/knqyf263/utern
# github.com/knqyf263/utern
../../go/src/github.com/knqyf263/utern/main.go:29:17: cannot use cli.StringFlag literal (type cli.StringFlag) as type cli.Flag in slice literal:
        cli.StringFlag does not implement cli.Flag (Apply method has pointer receiver)
../../go/src/github.com/knqyf263/utern/main.go:34:17: cannot use cli.StringFlag literal (type cli.StringFlag) as type cli.Flag in slice literal:
        cli.StringFlag does not implement cli.Flag (Apply method has pointer receiver)
../../go/src/github.com/knqyf263/utern/main.go:39:17: cannot use cli.StringFlag literal (type cli.StringFlag) as type cli.Flag in slice literal:
        cli.StringFlag does not implement cli.Flag (Apply method has pointer receiver)
../../go/src/github.com/knqyf263/utern/main.go:44:17: cannot use cli.StringFlag literal (type cli.StringFlag) as type cli.Flag in slice literal:
        cli.StringFlag does not implement cli.Flag (Apply method has pointer receiver)
../../go/src/github.com/knqyf263/utern/main.go:49:17: cannot use cli.StringFlag literal (type cli.StringFlag) as type cli.Flag in slice literal:
        cli.StringFlag does not implement cli.Flag (Apply method has pointer receiver)
../../go/src/github.com/knqyf263/utern/main.go:54:17: cannot use cli.StringFlag literal (type cli.StringFlag) as type cli.Flag in slice literal:
        cli.StringFlag does not implement cli.Flag (Apply method has pointer receiver)
../../go/src/github.com/knqyf263/utern/main.go:59:17: cannot use cli.StringFlag literal (type cli.StringFlag) as type cli.Flag in slice literal:
        cli.StringFlag does not implement cli.Flag (Apply method has pointer receiver)
../../go/src/github.com/knqyf263/utern/main.go:64:17: cannot use cli.StringFlag literal (type cli.StringFlag) as type cli.Flag in slice literal:
        cli.StringFlag does not implement cli.Flag (Apply method has pointer receiver)
../../go/src/github.com/knqyf263/utern/main.go:69:15: cannot use cli.BoolFlag literal (type cli.BoolFlag) as type cli.Flag in slice literal:
        cli.BoolFlag does not implement cli.Flag (Apply method has pointer receiver)
../../go/src/github.com/knqyf263/utern/main.go:73:15: cannot use cli.BoolFlag literal (type cli.BoolFlag) as type cli.Flag in slice literal:
        cli.BoolFlag does not implement cli.Flag (Apply method has pointer receiver)
../../go/src/github.com/knqyf263/utern/main.go:73:15: too many errors


❯ go version
go version go1.15.5 linux/amd64
```

--- a/README.md
+++ b/README.md
@@ -51,7 +51,9 @@ The log-group-query is a regular expression so you could provide "web-\w" to tai
 ## From source
 
 ```console
-$ go get -u github.com/knqyf263/utern
+$ git clone https://github.com/knqyf263/utern.git
+$ cd utern
+$ go install
 ```
 
 ## Binary (Including Windows)

